src/analytics/Properties.tsx

```diff
@@ -1579,7 +1580,7 @@ interface EarnEventsProperties {
   }
   [EarnEvents.earn_entrypoint_press]: undefined
   [EarnEvents.earn_add_crypto_action_press]: {
```

Comment:
should we use a different event? It's no longer just add crypto action. Or at the least, update the docs